PowerPoint_Playground/PowerPoint_Playground.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputPath>..\Build\</OutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <OutputPath>..\Build\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\PowerPoint_Adapter\PowerPoint_Adapter.csproj" />
+    <ProjectReference Include="..\PowerPoint_Engine\PowerPoint_Engine.csproj" />
+    <ProjectReference Include="..\PowerPoint_oM\PowerPoint_oM.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Adapter_oM">
+      <HintPath>..\..\..\ProgramData\BHoM\Assemblies\Adapter_oM.dll</HintPath>
```

Comment:
```suggestion
      <HintPath>C:\ProgramData\BHoM\Assemblies\Adapter_oM.dll</HintPath>
```

Reply:
That project is not supposed to be there anymore.
 File deleted